packages/paste-website/src/components/icons/TwilioIcon.tsx

```diff
@@ -13,16 +13,7 @@ const TwilioIcon = React.memo(({title = 'Twilio Icon', decorative = true, classN
   const uid = useUID();
   return (
     <div style={{color, width: size, height: size}} className={className}>
-      <svg
-        role="img"
-        aria-hidden={decorative}
-        aria-labelledby={uid}
-        className="prefix__twilio-mark"
-        fill="#F22F46"
-        viewBox="0 0 30 30"
-        width="100%"
-        height="100%"
-      >
+      <svg role="img" aria-hidden={decorative} aria-labelledby={uid} viewBox="0 0 30 30" width="100%" height="100%">
```

Comment:
👍 
Don't you have to pass that hex color in the sidebar though?

Reply:
Apparently not. Still totally works so I removed it